### ISSUE TITLE
Epic: Audit differences between code and documented field names

### ISSUE THREAD

**Issue by Neecolaa** [hackforla/peopledepot#432]
### Overview

There are field names in the code that do not currently match our documentation. We need to go through the existing django models, identify the mismatches, and update the code or documentation to comply with our desired naming conventions.

### Action Items

- [ ] List existing models
- [ ] For each model, create a table containing the different field names
- [ ] Verify desired naming conventions
- [ ] Update the documentation/code

#### 1.01 Code locations
   - 1.01.01 [/app/core/models.py](https://github.com/hackforla/peopledepot/blob/main/app/core/models.py)
   - 1.01.02 [/app/core/admin.py](https://github.com/hackforla/peopledepot/blob/main/app/core/admin.py)
   - 1.01.03 [/app/core/api/serializers.py](https://github.com/hackforla/peopledepot/blob/main/app/core/api/serializers.py)
   - 1.01.04 [/app/core/migrations/0002_user_current_job_title_user_current_skills_and_more.py](https://github.com/hackforla/peopledepot/blob/main/app/core/migrations/0002_user_current_job_title_user_current_skills_and_more.py)
  
#### 1.02 [People Depot Resources wiki page](https://github.com/hackforla/peopledepot/wiki/Resources-and-Links) for links
  - ERD
  - Table and Field Definitions
  - API Endpoint
- 1.03 [Spreadsheet, all fields](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=371053454#gid=371053454)

**Comment by fyliu** [comment]
I want to note that when we name a ForeignKeyField something like "user", Django will create a database column called "user_id". It's Django's convention to drop the "_id" from the end when converting from db to Django code.

I can find a link to a better description of this when I'm on a computer.

**Comment by Neecolaa** [comment]
`event` has a field called `brigade_id` that's commented out, but will need to be changed to `org_id`

**Comment by shmonks** [comment]
Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by ExperimentsInHonesty** [comment]
<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

1. Progress: We dicussed the need to get clarification from Fang on what we need to do next re the FK id in the comment above.  https://github.com/hackforla/peopledepot/issues/432#issuecomment-2463598500 @Neecolaa will follow up with him.
2. Blockers: time
3. Availability: monday meetings
4. ETA: we will never finish this 💀💀💀💀

**Comment by ExperimentsInHonesty** [comment]
We need en epic (unless their already is on, to organize these issues)
- https://github.com/hackforla/peopledepot/issues/436
- https://github.com/hackforla/peopledepot/issues/429
- https://github.com/hackforla/peopledepot/issues/172 - does not have table, might be old
- https://github.com/hackforla/peopledepot/issues/227 - does not have table, might be old


The issues may need to be redone to deal with any guidance we get from Fang re field names with "id" in them

**Comment by fyliu** [comment]
I think we should change that name in the comment. Those are supposed to be placeholders for relationships with models that don't exist yet. The code should be in the completed model, but we can't add them yet. But we also want to close the issue as complete, so we add them as comments.

These comments should only exist in models that we decided to have initially even though they depended on other models that didn't exist. So `User`, `Project`, `Event` I believe. Later create table issues used dependencies.

I see a couple of commented lines are probably safe to delete as well, like `day_of_week`, but we can also leave them for some audit issue before we release.

**Comment by ExperimentsInHonesty** [comment]
We decided the way to handle if a table creation has a FK, then add this text to the issue, and that will provide instructions on what the developer should do, to clear the error and make sure that their comments get uncommented when the FK table is created.

List of issues to add the text to
[Issues tab, label: table creation with FK in them](https://github.com/hackforla/peopledepot/issues?q=is%3Aissue%20state%3Aopen%20FK%20label%3A%22feature%3A%20table%20creation%22)

[PD: Table and field explanations, Tables](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=1228509670#gid=1228509670)


<details><summary>Example</summary>

See issue https://github.com/hackforla/peopledepot/issues/205

> ### Data Fields
> For each of the FK below if the FK table has not yet been created (i.e, the issue is still open), you will need to
>  - comment out the code after you create it 
>  - add an action item on the open issue to uncomment the code line you commented out when that issue's table is created. 
>  - Provide your file and line number as a permalink under resources in that issue
> 1. New fields
>     - [ ] (PK) id - int - Record Id
>     - [ ] (FK) project_id - int (https://github.com/hackforla/peopledepot/issues/16)
>     - [ ] (FK) stack_element_id - int - Record Id (#39 initial model, #204 modification)

    
</details> 


```
For each of the FK below if the FK table has not yet been created (i.e, the issue is still open), you will need to
 - comment out the code after you create it 
 - add an action item on the open issue to uncomment the code line you commented out when that issue's table is created. 
 - Provide your file and line number as a permalink under resources in that issue
```

**Comment by shmonks** [comment]
- @ExperimentsInHonesty and @Neecolaa, I copied the contents of [the comment above](https://github.com/hackforla/peopledepot/issues/432#issuecomment-2649406119) into a new issue, #472, and assigned it to myself

**Comment by shmonks** [comment]
Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by ExperimentsInHonesty** [comment]
- [ ] Make a new issue (put in icebox) review all comments after all tables are created
   - see Fang's comment  https://github.com/hackforla/peopledepot/issues/432#issuecomment-2617111894
     >I see a couple of commented lines are probably safe to delete as well, like `day_of_week`, but we can also leave them for some audit issue before we release.

**Comment by ExperimentsInHonesty** [comment]
@fyliu 
- [ ] Please revise the comment you left above https://github.com/hackforla/peopledepot/issues/432#issuecomment-2463598500

**Comment by ExperimentsInHonesty** [comment]
Next steps is to go through each item in column H, where the value in column J = AUDIT and determine if we are keeping or changing, and if changing, add the new name

**Comment by ExperimentsInHonesty** [comment]
We are almost done. There are only 43 left to check.... once we are done, we can review the code, and the open issues by table to see if they match or need their issues changed or update issues made.

**Comment by ExperimentsInHonesty** [comment]
Met with Del and we need to look at this
https://github.com/hackforla/peopledepot/pull/477#issuecomment-2864760038
https://github.com/hackforla/peopledepot/issues/461#issuecomment-2731223194

**Comment by ExperimentsInHonesty** [comment]
Asked for changes
- https://github.com/hackforla/peopledepot/issues/48#issuecomment-2874278097

**Comment by ExperimentsInHonesty** [comment]
look up each table and identify current titles.  If the table has not been made, edit the issue, otherwise it will get addressed in a change issue later.

**Comment by fyliu** [comment]
Here's the current generated descriptions of all the tables: [table-descriptions.csv](https://github.com/user-attachments/files/20620089/table-descriptions.csv)


<details>
  <summary>Instructions</summary>

```
# go to the git root directory
cd $(git rev-parse --show-toplevel)

# start psql in db container
./scripts/db.sh

# query all table descriptions and output to csv
\COPY (SELECT table_name, column_name, data_type, is_nullable
      FROM information_schema.columns
      WHERE table_schema = 'public'
      ORDER BY table_name, ordinal_position)
TO '/table-descriptions.csv' WITH (FORMAT CSV, HEADER);

# exit psql
<ctrl>+D

# copy csv file from db container to host
docker compose cp db:/table-descriptions.csv .

# the csv file is now written to the git root directory
```
</details>

I wrote a shell script to do all this so we don't have to look at the instructions anymore. I'll create a PR.
We can also create a GHA to update the csv file in the repo when PRs are merged. Maybe a DevOps issue?

**Comment by Neecolaa** [comment]
We're using [this table](https://docs.google.com/spreadsheets/d/1G9kDheY3MdITZnrthTFIUyprrTEvlMYpkFYWbCAMCmA/edit?usp=drive_link) to track our progress auditing the current names in code

**Comment by ExperimentsInHonesty** [comment]
- outstanding item has been added to agenda https://github.com/hackforla/peopledepot/issues/461#issuecomment-2997975570 to determine if we are renaming all the tables to not have underscores in their tites

**Comment by ExperimentsInHonesty** [comment]
fix all the timestamp types to be the same, csv from django calls them timestamp with time zone

**Comment by fyliu** [comment]
I know this is late, but there are model field attributes that might be useful for us to as documentation in the database. The [Field.db_comment](https://docs.djangoproject.com/en/5.2/ref/models/fields/#db-comment) looks like it'll show up for each column when we export the `.csv` database description. I think it also shows up in the Django admin site as guide text in the forms to explain what should go in the field.

**Comment by ExperimentsInHonesty** [comment]
yes, rename all the tables with no underscores

**Comment by Neecolaa** [comment]
We're currently going through all of the items in column J that still say AUDIT and making sure that the only ones that remain audit are tables that haven't been created yet. After this we will refine the formula that tells us what changes need to be made where.

**Comment by ExperimentsInHonesty** [comment]
screen shot from VRMS
<img width="538" height="530" alt="Image" src="https://github.com/user-attachments/assets/ee3272e4-c563-435c-bdc0-969cc7396a90" />

**Comment by ExperimentsInHonesty** [comment]
review [each item with audit](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=371053454#gid=371053454&fvid=1943590977) to determine if the table issue needs to be updated, or its been created since 2025-06-05

[table-descriptions as of 2025-06-05](https://docs.google.com/spreadsheets/d/1G9kDheY3MdITZnrthTFIUyprrTEvlMYpkFYWbCAMCmA/)

**Comment by Neecolaa** [comment]
Issue reference list for items with audit: 
- modern_job_title #62 
- org #56 
- permission_history #37 
- project_program_area_xref #45 
- project_stack_element_xref #205 
- project_url #41
- sdg_target_indicator #71
- soc_broad #72
- soc_detailed #73
- soc_minor #75
- url_status_type #384
- url_type #77
- user_availability #79
- user_check #80
- user_employment_change #81 
- win #82 
- win_type #61

**Comment by ExperimentsInHonesty** [comment]
We are keeping both is_active and userstatus.  is_active is required by Django,

we can make the value of is_active TRUE for all users, unless it's a duplicate account (e.g., they started with the wrong email, didn't do anything we need to save, and then opened a new account).

we will still use the userstatus field to indicate


id | name | description
-- | -- | --
1 | Inactive | Member who has not checked into their project or Community of Practice for 4 weeks and does not have a timeAwayHold
2 | Active | Member that is checking into meetings
3 | Time Away Hold | Hold placed by the member or their leader after they have announced a temporary absence
4 | Barred | Member who has been removed from the community

**Comment by Neecolaa** [comment]
For each issue above:

- check to see if all fields in spreadsheet are in issue and vice versa
- Reorder fields in spreadsheet so that they match the issue
- clear AUDIT labels
- check issue prioritization
- check issue off on list above

**Comment by ExperimentsInHonesty** [comment]
when we are done checking the issues (2 comments above), check formulas for next steps

**Comment by ExperimentsInHonesty** [comment]
@Neecolaa I met with Fang and updated the [spreadsheet for the events](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=749942651#gid=749942651).  Please review the cells with red in them, and determine their type

**Comment by Neecolaa** [comment]
the spreadsheet has been updated
start_time = datetime
video_conference_url = URLField
do you want me to update the erd with the new event stuff? I know event_occurrence is new, I imagine there's also changes to the other event tables

**Comment by ExperimentsInHonesty** [comment]
In the user availablity table we have day of week with is a FK to a day of week table.  This is used no other place.  If we should do it differently, we potentially can remove the day of week table.  Investigate. - Issue #79 would be affected.

**Comment by Neecolaa** [comment]
> [@Neecolaa](https://github.com/Neecolaa) I can't find the original issue for creating the permission table. Did it used to have a different name? Here is my search [is:issue "Table: Permission" label:"feature: table creation","feature: update table"](https://github.com/hackforla/peopledepot/issues?q=is%3Aissue%20%22Table%3A%20Permission%22%20label%3A%22feature%3A%20table%20creation%22%2C%22feature%3A%20update%20table%22)


@ExperimentsInHonesty 
the [tables tab in the spreadsheet](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=1228509670#gid=1228509670) has the original issue listed as #22. Looks like it was called user permission in the issue

**Comment by ExperimentsInHonesty** [comment]
Met with Fang, and we need to make the following changes

- https://github.com/hackforla/peopledepot/issues/88#issuecomment-3500095875

**Comment by ExperimentsInHonesty** [comment]
>In the user availablity table we have day of week with is a FK to a day of week table. This is used no other place. If we should do it differently, we potentially can remove the day of week table. Investigate.

skills and day of week, were originally going to be the purview of Civic Tech Jobs, but instead its going to be in the PeopleDepot database.  

This allows people to tell us their regular availbility per day of week and time of day, so that we can show them only projects that have collaboration session (aka meetings), that match their availability.

The skills will be used to help them find projects and contributions inside hackforla that match exiting skill levels and skills

Related 
- #79 
- #60

**Comment by ExperimentsInHonesty** [comment]
Review this issue to get it fixed and then reprioiritzed
- https://github.com/hackforla/peopledepot/issues/290

**Comment by ExperimentsInHonesty** [comment]
@Neecolaa rename table "org" to "organization" in ERD

**Comment by ExperimentsInHonesty** [comment]
@Fang why is the **project table** field: project_leads not showing up in the [table-descriptions from the application](https://docs.google.com/spreadsheets/d/1G9kDheY3MdITZnrthTFIUyprrTEvlMYpkFYWbCAMCmA/edit?gid=1268825842#gid=1268825842) even though it looks like it was added in the following
- https://github.com/hackforla/peopledepot/issues/607
- https://github.com/hackforla/peopledepot/pull/650

**Comment by ExperimentsInHonesty** [comment]
@fang why is the StackElement field: stack_element_type showing as element_type_id, when this issue and PR changed it. It needs to be stack_element_type_id or is it stackelementtype_id.
- https://github.com/hackforla/peopledepot/issues/576
- https://github.com/hackforla/peopledepot/pull/704

**Comment by ExperimentsInHonesty** [comment]
@Neecolaa 
- We originally were going to add the following fields to **user table** https://github.com/hackforla/peopledepot/issues/601
   - practice_area_secondary
   - practice_area_target_intake

It mentions that we moved it to another issue and this is the only issue I could find
- https://github.com/hackforla/peopledepot/issues/680

So we have to figure out what we do next.

**Comment by ExperimentsInHonesty** [comment]
@fyliu why are the following fields missing from the **wins table**, when they were added in the PR and Pull requests below.  I am using [table-descriptions from the application](https://docs.google.com/spreadsheets/d/1G9kDheY3MdITZnrthTFIUyprrTEvlMYpkFYWbCAMCmA/edit?gid=1268825842#gid=1268825842) to identify they ere missing

- field names
   - practice_areas
   - teams
- Issue and PR
   - https://github.com/hackforla/peopledepot/issues/82
   - https://github.com/hackforla/peopledepot/pull/591

**Comment by fyliu** [comment]
@ExperimentsInHonesty I'm seeing those fields in the code, but not in the spreadsheet. I'll re-run the export and import scripts to see if there's a step that didn't carry them over or if I imported an outdated table-descriptions file on 09/01/2026.

**Update**: I see. It is updated. Those fields are many-to-many relationships, and django made its own xref tables for them called `core_win_practice_areas` and `core_win_teams`. I guess we should make xref tables for them if we want to keep track of the create and update timestamps.

**Comment by ExperimentsInHonesty** [comment]
We have an xref table that is formatted differently than the others.  It dosent have 
- uuid
- created_at
- updated_at

and it has a field called
- id

This is what is in each of our files

spreadsheet: [table-descriptions from the application, table-descriptions 2026-09-01](https://docs.google.com/spreadsheets/d/1G9kDheY3MdITZnrthTFIUyprrTEvlMYpkFYWbCAMCmA/edit?gid=1268825842#gid=1268825842)
table_name-prefix | column_name | data_type
-- | -- | --
user_practice_area_target_intake | id | bigint
user_practice_area_target_intake | user_id | uuid
user_practice_area_target_intake | practicearea_id | uuid

spreadsheet: [PD: Table and field explanations, current Field Permissions](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=749942651#gid=749942651)
Table Name | column_name | data_type
-- | -- | --
UserPracticeAreaTargetIntakeXref | uuid | int
UserPracticeAreaTargetIntakeXref | created_at | DateTimeField
UserPracticeAreaTargetIntakeXref | updated_at | DateTimeField
UserPracticeAreaTargetIntakeXref | user | int
UserPracticeAreaTargetIntakeXref | practice_area | int

ERD: [People Depot Schema ERD](https://dbdiagram.io/d/People-Depot-Schema-ERD-68917d4edd90d178656e4061)
<img width="324" height="204" alt="Image" src="https://github.com/user-attachments/assets/0bc10def-e882-4627-8744-106248298d6d" />


But there is another xref file and here is what it looks like in each of our files

spreadsheet: [table-descriptions from the application, table-descriptions 2026-09-01](https://docs.google.com/spreadsheets/d/1G9kDheY3MdITZnrthTFIUyprrTEvlMYpkFYWbCAMCmA/edit?gid=1268825842#gid=1268825842)
table_name-prefix | column_name | data_type
-- | -- | --
project_affiliate_xref | uuid | uuid
project_affiliate_xref | created_at | timestamp with time zone
project_affiliate_xref | updated_at | timestamp with time zone
project_affiliate_xref | ended_at | timestamp with time zone
project_affiliate_xref | is_sponsor | boolean
project_affiliate_xref | is_partner | boolean
project_affiliate_xref | affiliate_id | uuid
project_affiliate_xref | project_id | uuid


spreadsheet: [PD: Table and field explanations, current Field Permissions](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=749942651#gid=749942651)
Table Name | column_name | data_type
-- | -- | --
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | uuid | int
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | created_at | DateTimeField
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | updated_at | DateTimeField
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | affiliate_id | int
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | ended | Date
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | is_sponsor | boolean
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | is_partner | boolean
ProjectAffiliateXref(Formerly ProjectSponsorPartnerXref) | project_id | int


ERD: [People Depot Schema ERD](https://dbdiagram.io/d/People-Depot-Schema-ERD-68917d4edd90d178656e4061)
<img width="293" height="299" alt="Image" src="https://github.com/user-attachments/assets/4f422911-c7c0-4461-8d3b-b73bd60ec1a9" />

**Comment by fyliu** [comment]
Right. When we make/define our own xref tables, they have the `uuid`, `created_at`, `updated_at` like all our other tables. When we let Django make them, it just contains a numeric `id` and FKs to the tables. We usually make our own so we can store more metadata about the relationships if we need to.

**Comment by ExperimentsInHonesty** [comment]
Fang said:
>When we make/define our own xref tables, they have the uuid, created_at, updated_at like all our other tables. When we let Django make them, it just contains a numeric id and FKs to the tables. We usually make our own so we can store more metadata about the relationships if we need to.

So let's discuss the following: 

### Tables Django has created but that we don't have in the spreadsheet: [PD: Table and field explanations, current Field Permissions](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=749942651#gid=749942651).  The question is if we are going to create tables ourselves
- project_project_leads
- user_groups
- user_user_permissions
- win_practice_areas
- win_teams

### Tables that Django has created that we have in the spreadsheet: [PD: Table and field explanations, current Field Permissions](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=749942651#gid=749942651).  We will need to create these tables with new issues


### user_practice_area_secondary

table name | column_name | data_type
-- | -- | --
UserPracticeAreaSecondaryXref | uuid | int
UserPracticeAreaSecondaryXref | created_at | DateTimeField
UserPracticeAreaSecondaryXref | updated_at | DateTimeField
UserPracticeAreaSecondaryXref | user | int
UserPracticeAreaSecondaryXref | practice_area | int


### user_practice_area_target_intake

table name | column_name | data_type
-- | -- | --
UserPracticeAreaTargetIntakeXref | uuid | int
UserPracticeAreaTargetIntakeXref | created_at | DateTimeField
UserPracticeAreaTargetIntakeXref | updated_at | DateTimeField
UserPracticeAreaTargetIntakeXref | user | int
UserPracticeAreaTargetIntakeXref | practice_area | int

**Comment by ExperimentsInHonesty** [comment]
I reviewed the [PD: Table and field explanations, current Field Permissions](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=749942651#gid=749942651) to see what the status of the outstanding issues is

Issues that are blocked by other issues
- https://github.com/hackforla/peopledepot/issues/37 Fang made this issue dependent on https://github.com/hackforla/peopledepot/issues/181. It will potentially be useful for 2 or more models histories
	- we do need to block #37 because it might be managed django-auditlog instead of us having to add and manage a history table.
- https://github.com/hackforla/peopledepot/issues/67 is blocked by https://github.com/hackforla/peopledepot/issues/181
- https://github.com/hackforla/peopledepot/issues/79 is blocked by #88 because user availability and day of week are intertwined.  We should sort those out together.

Closed Issues with problems
- project table has a problem, see https://github.com/hackforla/peopledepot/issues/432#issuecomment-5543658376
    - We need to **make our own xref table** for project_leads. It's okay that the fields involving xref tables don't show up in the Project or other tables.
    	- [ ] project_leads #712 (still needs adding to ERD/SS)
- StackElement has a problem, see https://github.com/hackforla/peopledepot/issues/432#issuecomment-5544479670
	- this **change is present** in the newest tables export from the DB. 
- User has a problem, see https://github.com/hackforla/peopledepot/issues/432#issuecomment-5545277747
	- same as above with **needing our own xref tables** for these fields
		- [x] practice_area_secondary #680
		- [x] practice_area_target_intake #711 
- Wins has a problem, see https://github.com/hackforla/peopledepot/issues/432#issuecomment-5545519377
	- same as above with **needing our own xref tables** for these fields
		- [ ] practice_areas #713 (still needs adding to ERD/SS)
		- [ ] teams

Issues assigned
- https://github.com/hackforla/peopledepot/issues/605 assigned to Esmeralda
- https://github.com/hackforla/peopledepot/issues/606 currently assigned to Roberto

Issues not yet assigned
- https://github.com/hackforla/peopledepot/issues/604
- https://github.com/hackforla/peopledepot/issues/575

Issues that will need to be checked once the table export is done again
- #577 
	- `created_by` and `updated_by` are added as commented out code because issue #172 is still open. Coder was following directions in the issue. But I don't see it as a blocker for uncommenting that code.

No issues have been made for the following tables
- UserIntakeTargetSkillXref
- UserPracticeAreaSecondaryXref
- UserPracticeAreaTargetIntakeXref

**Comment by fyliu** [comment]
We need to work on/verify the fields for **PracticeArea** and **ProgramArea**. The `project_program_area_status_type` field in PracticeArea definitely should be in ProgramArea. 
- The issue had the problem: https://github.com/hackforla/peopledepot/issues/608
- It's probably a copy/paste mistake when we worked through it. https://docs.google.com/spreadsheets/d/1G9kDheY3MdITZnrthTFIUyprrTEvlMYpkFYWbCAMCmA/edit?gid=1776135945#gid=1776135945

<img width="889" height="472" alt="Image" src="https://github.com/user-attachments/assets/510928db-93fa-48ab-a0cd-a153f0a200af" />

**Comment by Neecolaa** [comment]
Issues have been created for 4/5 xref problems. 

- Would `WinTeamsXref` be preferred for the many-to-many relationship between Win and Project?
- I marked the create table issues that were ready to be prioritized as done, but do we also need to make update issues for their associated tables?
